dl/src/main/scala/com/intel/analytics/bigdl/utils/File.scala

```diff
@@ -49,28 +57,84 @@ object File {
   }
 
   /**
-   * Save scala object into a Java object file
+   * Save scala object into a local/hdfs path
    *
-   * @param obj obj to be saved.
-   * @param fileName file name saved to.
+   * @param obj object to be saved.
+   * @param outputPath local/hdfs output path.
    * @param isOverwrite if overwrite.
    */
-  def save(obj: Serializable, fileName: String, isOverwrite: Boolean = false): Unit = {
-    if (Files.exists(Paths.get(fileName)) && !isOverwrite) {
-      throw new RuntimeException("file exists!")
+  def save(obj: Serializable, outputPath: String, isOverwrite: Boolean = false): Unit = {
+    var localFilePath: String = outputPath
+    if (outputPath.startsWith(File.localfsPrefix)) {
+      localFilePath = outputPath.substring(File.localfsPrefix.length)
+      if (Files.exists(Paths.get(outputPath)) && !isOverwrite) {
+        throw new RuntimeException(s"file ${outputPath} already exists!")
+      }
+    } else {
+      localFilePath = System.getProperty("user.dir") + "/output-model.tmp"
```

Comment:
`System.getProperty("user.dir")` gets the directory where `java` was run from, so this will always works with any env with java installed. Right?